### ISSUE TITLE
How to access certain elements 

### ISSUE THREAD

**Issue by AryaBaggins** [appium/appium-mac2-driver#97]
Hello, I have a silly question and this is by no means an appium issue but I still hope to get some answers. I've been trying to do some macos app automation testing and I'm having a hard time inspecting the elements of the target apps using accessibility inspector. It appears that I can only inspect the native apps of mac for example, textedit, but when I try to inspect like a published app from a third party, like intellij, it only gives the locator of the entire frame/cell of the app, which makes it impossible to write automation scripts with appium. Is there some sort of security settings I need to make?  Thanks!!!

**Comment by KazuCocoa** [comment]
This driver depends on how https://developer.apple.com/documentation/xctest works.
For example, if an app did not use native element code to build its UI, perhaps this driver does not work since the XCTest framework cannot detect them.

I don't know well what security preference etc affects the detection, but what about enabling _speak selection_ in System preference -> Accessibility -> Spoken Content?
On iOS, the item extends element detection as OS system. I don't know this helps on macOS as well.

**Comment by aakrandan** [comment]
@AryaBaggins did you manage to fix your issue? if not perhaps i can help!

**Comment by TLPP30** [comment]
@aakrandan no, turns out the app I was trying to test was not a native xcode project and it has its own ui framework. Did you happen to have to deal with something like that ?

**Comment by Wildprogrammer** [comment]
@aakrandan Is there any solution to this problem?
